source/documentation/monitoring_apps/metrics.md

```diff
@@ -2,94 +2,56 @@
 
 ## Metrics
 
-Cloud Foundry provides time series data, or metrics, for each instance of your PaaS app. You can receive, store and view this data in a monitoring system of your choice by either:
+Cloud Foundry provides time series data, or metrics, for each instance of your PaaS app. You can receive, store and view this data in a monitoring system of your choice by deploying one of the following apps:
 
-- using the [Prometheus](https://prometheus.io/) [external link] endpoint provided by the GOV.UK PaaS team
-- deploying the `paas-metric-exporter` app to push metrics data in [StatsD](https://github.com/etsy/statsd/wiki) [external link] format
+- the [`paas-prometheus-exporter`](https://github.com/alphagov/paas-prometheus-exporter)
```

Comment:
I would add here:
```
- the [`paas-prometheus-exporter`](https://github.com/alphagov/paas-prometheus-exporter): to collect and expose metrics to be consumed [in the prometheus format](https://prometheus.io/docs/introduction/overview/)
```

Reply:
Usually "Prometheus" consumes (or scrapes) the metrics. https://codeblog.dotsandbrackets.com/scraping-application-metrics-prometheus/